src/index.js

```diff
@@ -35,8 +47,31 @@ class GeoRaster {
     if (debug) console.log('this after construction:', this);
   }
 
+  async getValues(left, top, right, bottom, width, height) {
+    console.log('getValues: ', left, top, right, bottom, width, height);
+    // note this.image and this.geotiff both have a readRasters method;
+    // they are not the same thing. use this.geotiff for experimental version
+    // that reads from best overview
+    let data = await this.geotiff.readRasters({
+      window: [left, top, right, bottom],
+      width: width,
+      height: height,
+      resampleMethod: 'bilinear',
+    });
+
+    return data;
+  }
+
+
+  async initialize(debug) {
+    if (this._url) {
+      // initialize these outside worker to avoid weird worker error
+      // I don't see how cache option is passed through with fromUrl,
+      // though constantinius says it should work: https://github.com/geotiffjs/geotiff.js/issues/61
```

Comment:
Thank you for the comment / explanation.  We could also consider caching on the georaster-side by memoizing our getValues function?  Would be useful in the future when NetCDF, JPEG200 and other formats are added in. Definitely not a requirement for this phase though.